src/plugins/external_tool_support/src/blast/AlignToReferenceBlastWorker.cpp

```diff
@@ -345,23 +358,27 @@ SharedDbiDataHandler AlignToReferenceBlastTask::getAnnotations() const {
 }
 
 QList<QPair<QString, QPair<int, bool>>> AlignToReferenceBlastTask::getAcceptedReads() const {
+    CHECK(blastTask != nullptr, {});
```

Comment:
Does it possible for `blastTask` be equal to `nullptr` here? If it's not, it should be `SAFE_POINT`

Reply:
Yes, SAFE_POINT will be better.